packages/core/src/utilities/color/_color.docs.scss

```diff
@@ -1,89 +1,30 @@
 /*
-Color
-
-Use the color utility to change an element's text color.
-
-The options below offer color palette combinations that fall within the range of Section 508 compliant foreground/background color contrast ratios. To ensure that text remains accessible, **use only these permitted color combinations**.
-
-**Format**: `ds-u-color--[COLOR NAME]-[SHADE]`
-
-@hide-markup
-
-Markup:
-<% var groups = {
-  'white': [
-    'base',
-    'black',
-    'muted',
-    'gray',
-    'primary', 'primary-darker', 'primary-darkest',
-    'error', 'error-dark',
-    'success'
-  ],
-  'gray-lightest': ['muted', 'primary', 'error-dark'],
-  'gray-lighter': ['base', 'primary-darker'],
-  'gray-light': ['base', 'primary-darkest'],
-  'background-inverse': ['base-inverse', 'muted-inverse', 'error-light'],
-  'base': ['white'],
-  'gray-dark': ['white'],
-  'gray': ['white'],
-  'primary-darkest': ['white'],
-  'primary-darker': ['white'],
-  'primary': ['white'],
-  'primary-alt-darkest': ['white'],
-  'primary-alt-dark': ['base'],
-  'primary-alt': ['base'],
-  'green': ['white'],
-  'success': ['white'],
-  'green-light': ['base'],
-  'success-light': ['base'],
-  'gold': ['base'],
-  'warn': ['base'],
-  'gold-light': ['base'],
-  'warn-light': ['base'],
-  'secondary-darkest': ['white'],
-  'error-darkest': ['white'],
-  'secondary-dark': ['white'],
-  'error-dark': ['white'],
-  'secondary': ['white'],
-  'error': ['white'],
-  'primary-alt-lightest': ['base'],
-  'green-lighter': ['base'],
-  'success-lighter': ['base'],
-  'green-lightest': ['base'],
-  'success-lightest': ['base'],
-  'gold-lighter': ['base'],
-  'warn-lighter': ['base'],
-  'gold-lightest': ['base'],
-  'warn-lightest': ['base'],
-  'secondary-lightest': ['base'],
-  'error-lightest': ['base'],
-}; -%>
-<% Object.keys(groups).forEach(fill => { -%>
-<% groups[fill].forEach(color => { -%>
-  <div class="ds-u-color--<%= color %> ds-u-fill--<%= fill %> ds-u-padding--1">
-    <code class="ds-h3 ds-u-fill--transparent ds-u-color--<%= color %>">ds-u-color--<%= color %></code>
-    <br />
-    <span class="ds-u-font-size--small">on</span>
-    <code class="ds-u-fill--transparent ds-u-color--<%= color %>">ds-u-fill--<%= fill %></code>
-  </div>
-<% }) -%>
-<% }) %>
-
-Style guide: utilities.color
-*/
-
-/*
+Text color
```

Comment:
What do you think about also renaming all the files from `color` to `text-color`?

Reply:
I like that idea!